inc/utils/content.php

```diff
@@ -564,17 +581,19 @@ function get_categories_array($post){
 
  function odm_echo_extras($postID = "") {
  	 $postID = $postID ? $postID : get_the_ID();
-	 if (function_exists('get_post_meta')) :
-		 $get_author = get_post_meta($postID, 'author', true);
-		 $get_localized_author = get_post_meta($postID, 'author_'.odm_language_manager()->get_current_language(), true);
-	   if ($get_author != '' || $get_localized_author != ''):
-	     $news_source_info = '<span class="lsf">&#xE041;</span> ';
-	     if ($get_localized_author != ''):
-	         $news_source_info .= $get_localized_author.'<br />';
-	     else:
-	         $news_source_info .= $get_author.'<br />';
-	     endif;
-	   endif;
+	 if (function_exists('get_post_meta')):
```

Comment:
@Huyeng The UI elements should be kept consistent across the site. It would be confusing for the user if the news articles feature the author ( plus link ) on the bottom of the article while the other content types feature it on the metadata area (without link).

Can you come up with a solution which is consistent in terms of UX (User experience)